lib/engine/config/game/g_18_co.rb

```diff
@@ -0,0 +1,1260 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/RedundantCopDisableDirective, Layout/LineLength, Layout/HeredocIndentation
+
+module Engine
+  module Config
+    module Game
+      module G18CO
+        JSON = <<-'DATA'
+{
+	"filename": "18_co",
+	"modulename": "18CO",
+	"currencyFormatStr": "$%d",
+	"bankCash": 10000,
+	"certLimit": {
+		"3": 19,
+		"4": 14,
+		"5": 12,
+		"6": 10
+	},
+	"startingCash": {
+		"3": 500,
+		"4": 375,
+		"5": 300,
+		"6": 250
+	},
+	"capitalization": "incremental",
+	"layout": "pointy",
+	"axes": {
+		"rows": "numbers",
+		"columns": "letters"
+	},
+	"mustSellInBlocks": false,
+	"locationNames": {
+		"A11": "Laramie, WY",
+		"A17": "Cheyenne, WY",
+		"B10": "Walden",
+		"B22": "Sterling",
+		"B26": "Lincoln, NE (SLC +100)",
```

Comment:
Wonder if this makes the map messy? An alternative could be to add an ability to SLC that gives this bonus.
(Hm, I do not know what SLC is....)

Reply:
Salt Lake City. I was following the example set by 1882; ( HB +100 ) on the two hexes signifying that a connection to Hudson Bay gives a +100 bonus.

Reply:
Here is what it looks like on the map.

<img width="151" alt="Screen Shot 2020-10-12 at 5 54 02 PM" src="https://user-images.githubusercontent.com/15675400/95800158-f724de80-0cb3-11eb-80d3-1ba6344820fc.png">